src/components/Files/FileItemContextMenu.tsx

```diff
@@ -0,0 +1,73 @@
+import { IconDownload, IconTrashX } from '@tabler/icons-react';
+import { MouseEventHandler } from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import classNames from 'classnames';
+
+import { DialFile } from '@/src/types/files';
+
+import { stopBubbling } from '@/src/constants/chat';
+
+import DotsIcon from '../../../public/images/icons/dots-vertical.svg';
+import { Menu, MenuItem } from '../Common/DropdownMenu';
+
+interface ContextMenuProps {
+  file: DialFile;
+  className: string;
+  onDelete: MouseEventHandler<unknown>;
+}
+
+export const FileItemContextMenu = ({
+  file,
+  className,
+  onDelete,
+}: ContextMenuProps) => {
+  const { t } = useTranslation('sidebar');
+  return (
+    <>
+      <Menu
+        type="contextMenu"
+        trigger={
+          <DotsIcon
+            className={classNames('rotate-90 text-gray-500')}
+            width={18}
+            height={18}
+            size={18}
+          />
```

Comment:
1) You don't need to use `classNames` here
2) why we still use `DotsIcon` with rotation instead of `IconDots` from `@tabler/icons-react`? You won't find difference
```suggestion
          <IconDots
            className="text-gray-500"
            size={16}
          />
```